setup.py

```diff
@@ -1,4 +1,15 @@
+import sys
 from setuptools import setup, find_packages
+from distutils.command.install import install
+from distutils.command.build import build
+from subprocess import call
+
+class Build(build):
+    def run(self):
+        cmd = ["make", "-C", "./SJARACNe"]
+        if call(cmd) != 0:
+            sys.exit(-1)
```

Comment:
If make fails, would this show the error message?

Reply:
It does show error messages (I just tried running it without a Makefile to double check). Python calls on `make -C ./SJARACNe` on the command line, so any issue with make when running `python setup.py build` is printed on the command line.